app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -473,10 +488,18 @@ private void attachToWindow(@NonNull WindowWidget aWindow, @Nullable WindowWidge
         mKeyboard.attachToWindow(aWindow);
         mTray.attachToWindow(aWindow);
 
+        if (mTabsBar != null) {
+            mTabsBar.attachToWindow(aWindow);
+        }
+
         if (aPrevWindow != null) {
             updateWidget(mNavigationBar);
+            updateWidget(mTabsBar);
```

Comment:
This is a mistake. You have the good one bellow with the null check

Reply:
Done.